dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -167,7 +173,13 @@ public void handle(TaskCallBack taskCallBack) throws TaskException {
 
     @Override
     public void cancel() throws TaskException {
-
+        String type = sqlParameters.getType();
+        if (DbType.HIVE == DbType.valueOf(type)) {
+            List<String> appIds = ProcessUtils.killYarnJob(taskExecutionContext);
```

Comment:
Can we move this in `org.apache.dolphinscheduler.plugin.task.api.ProcessUtils` 
cc @caishunfeng

Reply:
Can I write a method of kill yarn's job in the [task-api] ? No available tool classes found in [task-plugin]